Foundation/NSTimeZone.swift

```diff
@@ -197,8 +197,19 @@ extension NSTimeZone {
     
     public class func knownTimeZoneNames() -> [String] { NSUnimplemented() }
     
-    public class func abbreviationDictionary() -> [String : String] { NSUnimplemented() }
-    public class func setAbbreviationDictionary(dict: [String : String]) { NSUnimplemented() }
+    public class func abbreviationDictionary() -> [String : String] {
+        let dictionary = CFTimeZoneCopyAbbreviationDictionary()._nsObject
```

Comment:
actually this return value is a singleton stored value iirc.

effectively it is 
```swift
[
    "ADT" : "America/Halifax",
    "AKDT" : "America/Juneau",
    "AKST" : "America/Juneau",
    "ART" : "America/Argentina/Buenos_Aires",
    "AST" : "America/Halifax",
    "BDT" : "Asia/Dhaka",
    "BRST" : "America/Sao_Paulo",
    "BRT" : "America/Sao_Paulo",
    "BST" : "Europe/London",
    "CAT" : "Africa/Harare",
    "CDT" : "America/Chicago",
    "CEST" : "Europe/Paris",
    "CET" : "Europe/Paris",
    "CLST" : "America/Santiago",
    "CLT" : "America/Santiago",
    "COT" : "America/Bogota",
    "CST" : "America/Chicago",
    "EAT" : "Africa/Addis_Ababa",
    "EDT" : "America/New_York",
    "EEST" : "Europe/Istanbul",
    "EET" : "Europe/Istanbul",
    "EST" : "America/New_York",
    "GMT" : "GMT",
    "GST" : "Asia/Dubai",
    "HKT" : "Asia/Hong_Kong",
    "HST" : "Pacific/Honolulu",
    "ICT" : "Asia/Bangkok",
    "IRST" : "Asia/Tehran",
    "IST" : "Asia/Calcutta",
    "JST" : "Asia/Tokyo",
    "KST" : "Asia/Seoul",
    "MDT" : "America/Denver",
    "MSD" : "Europe/Moscow",
    "MSK" : "Europe/Moscow",
    "MST" : "America/Denver",
    "NZDT" : "Pacific/Auckland",
    "NZST" : "Pacific/Auckland",
    "PDT" : "America/Los_Angeles",
    "PET" : "America/Lima",
    "PHT" : "Asia/Manila",
    "PKT" : "Asia/Karachi",
    "PST" : "America/Los_Angeles",
    "SGT" : "Asia/Singapore",
    "UTC" : "UTC",
    "WAT" : "Africa/Lagos",
    "WEST" : "Europe/Lisbon",
    "WET" : "Europe/Lisbon",
    "WIT" : "Asia/Jakarta"
]
```